Qube TechRadar.csv

```diff
@@ -1,7 +1,7 @@
 name,ring,quadrant,isNew,description
 Go,Adopt,languages-and-frameworks,FALSE,"Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
 Visual Studio Code,Trial,Tools,FALSE,"Visual Studio Code is a code editor redefined and optimized for building and debugging modern web and cloud applications."
-AWS DevOps,Trial,Platforms,FALSE,"AWS DevOps provides services to simplify provisioning and managing infrastructure, deploying application code, automating software release processes, and monitoring your application and infrastructure performance."
+Yocto Linux,Adopt,Platforms,FALSE,"Custom linux distribution based on Yocto Project."
```

Comment:
Do we need `Yocto Linux` as a separate entry, when we have `Yocto Project`?  Maybe we could just remove the `AWS DevOps` entry?